scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -27,10 +27,6 @@ Names:
 [195..203): classTag => _root_.scala.reflect.package.classTag(Lscala/reflect/ClassTag;)Lscala/reflect/ClassTag;.
 [204..207): Int => _root_.scala.Int#
 
-Messages:
-[41..47): [warning] Unused import
```

Comment:
I expected this message to be persisted with the first document along with all the names. Only messages that are reported after typer (like deprecation) should be in the second document. That would mean that a single document is the norm.

Reply:
I was curious myself why messages weren’t persisted in the phase they were created but don’t understand enough about how semantic db works yet to answer the question.